### DISCUSSION TITLE
申请退款快2个月了，还没退，不知缘由把我case关了！！！

### DISCUSSION THREAD

**Discussion by vincentye9682** [getlantern/lantern-forum-cn#1100] [General]
邮箱: [yizunquan@163.com](mailto:yizunquan@163.com)

Payment time: 2024-11-07 11:20:28

Payment method: CMB Credit Card(7419)

游戏充值订单号: P01202411071119396916555

Full name of payee: 广州饶铭网络科技有限公司

Order No.: 2024110722001476771458238537

Merchant order No.: 2024110711194145426

![391765796-a79c6447-e78b-48c3-9b67-d7b9f7b5528f](https://github.com/user-attachments/assets/b9e1894f-0b34-43e4-ae87-ade2befbcf81)
![111](https://github.com/user-attachments/assets/bec9466c-a27c-41ff-b852-5bb951a2bd42)
![222](https://github.com/user-attachments/assets/4b5e87a2-dc39-4d2b-bd7b-d4f3e9b279b7)

退款！！！退款！！！退款！！！

**Comment by MaxMKMax** [comment]
@vincentye9682 您好. 我估计系统自动关了. 我帮您重新提交看看. 您之前又得到customer support的任何回复吗?

**Reply by vincentye9682** [reply]
@Mike777Khoo 你好，没有收到过任何customer support 的回复，唯一收到的就是说上面图片中所示case关闭

**Reply by MaxMKMax** [reply]
@vincentye9682 您好. 非常抱歉看来您的情况我们的系统无法给您退款. 原因是您的付费方式因为国际条约我们这里弄不了. 需要您和银行或信用卡哪里申请退款. 您可以用这个单据和他们申请.

真的非常抱歉啊. 需要麻烦您了. 我们的支付部门是这么回复您的.

**Reply by vincentye9682** [reply]
@Mike777Khoo 这跟支付方式/国际条约有什么关系？银行都说这钱已经转出去了，是没办法退回的，只能跟你们申请退款，你们收了款现在不退，这种理由是不是太敷衍了？

**Reply by MaxMKMax** [reply]
@vincentye9682 您好. 因为我们无法直接收款所以和中国的第三方合作收款. 所以我们的系统这里弄不了啊. 其实我们的购买协议也有些购买后是不退款的. 只是我们也想让你们的体验好所以尽量给你们退款. 我帮您问问看还有什么方法可以退款吗. 请您耐心等待吧. 谢谢您的谅解!

**Reply by vincentye9682** [reply]
@Mike777Khoo 好的，麻烦帮我处理一下，谢谢，已经2个多月了

**Reply by vincentye9682** [reply]
@Mike777Khoo 怎么样了？我现在登录也登录不了，你们说退款退不了，我现在想再登录账户也登录不上，收不到验证码邮件。。。。真是服务也没使上，钱也不给退，你们这也是坑到家了
